touchdown/ssh/client.py

```diff
@@ -96,6 +120,15 @@ def verify_transport(self):
         # but rubbish for developers D:
         return
 
+    def get_input_encoding(self):
+        result_buf = six.StringIO()
+        transport = self.get_transport()
+        self._run(transport, 'printenv LANG', stdout=result_buf)
+        result_buf.seek(0)
+        lang = result_buf.read()
+        if '.' in lang:
+            self.input_encoding = lang.rsplit('.', 1)[-1]
+
```

Comment:
I was wondering if there was a better paramiko API for this (our _run method is meant for streaming logs as they happen rather than being used like subprocess's check_output), alas the exec_command method looks a bit rubbish. So let's keep it like this :+1: